index.js

```diff
@@ -262,13 +255,34 @@ module.exports = {
     'selector-max-type': 2,
     'selector-max-universal': 1,
     'selector-no-qualifying-type': true,
-    'selector-no-vendor-prefix': true,
-    'shorthand-property-no-redundant-values': true,
-    'string-quotes': 'double',
     'unicode-bom': 'never',
     'value-list-comma-newline-after': 'never-multi-line',
     'value-list-comma-newline-before': 'never-multi-line',
     'value-list-comma-space-after': 'always',
-    'value-no-vendor-prefix': true
-  }
+  },
+  'overrides': [
+    {
+      'files': '**/*.scss',
+      'extends': ['stylelint-config-standard-scss'],
```

Comment:
Previously, we inherited rules from the css config. Now, we don't, right?

Reply:
With the new overrides syntax, you automatically get the CSS rules (the ones also defined in this file). However, I do realize now that you used to extend the SCSS rules, _then_ the CSS ones. Might have to restructure things a bit, but should still be possible to just have a single file for the user to extend at least. Will try to fix.

Reply:
TBH, whatever makes more sense and it's easier to maintain. I mean, sure the next version will be a major bump, but still we try to keep breaking changes to the minimum :)

Thanks for working on this!

Reply:
yeah fair enough, I'm trying something rn but no matter whether it works or not we could always just go back to not using `overrides` and having to separate imports. But we'll see how it goes. There's no way to run the tests locally is there?

Reply:
nvm, just cloned the bootstrap repo manually (might want to refactor the CI steps into a separate script)

Reply:
Alright so this didn't decrease the amount of errors, but it's still the proper way to do it. Will mark this particular issue as resolved for now, we can track/discuss the errors in the general thread.